programs/vote/src/vote_processor.rs

```diff
@@ -1096,6 +1186,253 @@ mod tests {
         );
     }
 
+    fn perform_authorize_with_seed_test(
+        authorization_type: VoteAuthorize,
+        vote_pubkey: Pubkey,
+        vote_account: AccountSharedData,
+        current_authority_base_key: Pubkey,
+        current_authority_seed: String,
+        current_authority_owner: Pubkey,
+        new_authority_pubkey: Pubkey,
+    ) {
+        let clock = Clock {
+            epoch: 1,
+            leader_schedule_epoch: 2,
+            ..Clock::default()
+        };
+        let clock_account = account::create_account_shared_data_for_test(&clock);
+        let transaction_accounts = vec![
+            (vote_pubkey, vote_account),
+            (sysvar::clock::id(), clock_account),
+            (current_authority_base_key, AccountSharedData::default()),
+        ];
+        let mut instruction_accounts = vec![
+            AccountMeta {
+                pubkey: vote_pubkey,
+                is_signer: false,
+                is_writable: false,
```

Comment:
```suggestion
                is_writable: true,
```
Right?

Reply:
Ha! I think the stake precedent is wrong, but also realizing T/F here doesn't really matter, since account writability violations are tracked outside of process_instruction